Gaming the Syllabus/Assets/Scripts/Portal.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Portal : MonoBehaviour
+{
+    public string levelName;
```

Comment:
Standard practice is to keep all variables private for encapsulation and add the [SerializeField] attribute if it needs to be visible in the inspector

Reply:
quick question, so "[SerializeField]" simply let you change/set values from the inspector?

Reply:
Technically it forces Unity to serialize a private field. I'm unfamiliar with the specifics, but it basically just makes a private field show up in the inspector. Public fields show up there by default